_includes/tutorials/serialization/kstreams/code/build.gradle

```diff
@@ -4,63 +4,89 @@ buildscript {
   }
   dependencies {
     classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.15.1"
+    classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.12"
   }
 }
 
 plugins {
   id "java"
   id "application"
-  id "com.google.cloud.tools.jib" version "1.1.1"
+  id "idea"
+  id "com.google.cloud.tools.jib" version "2.4.0"
   id "com.github.johnrengelman.shadow" version "5.1.0"
+  id "com.google.protobuf" version "0.8.12"
 }
 
-sourceCompatibility = "1.8"
-targetCompatibility = "1.8"
+sourceCompatibility = "11"
```

Comment:
Why Java 11?  Overall I'm ok with it, but I think we want to be consistent across all tutorials.

Reply:
yes, I will propose that we will move to java 11 across all tutorials. 

* java 11 was supported in AK for a while
* java 8 is sunsetting 
* I wanted to use this tutorial to validate integration with CI/DC

Reply:
Also, I wanted to use `Map.of` `List.of` helper methods without bringing extra libraries.

Reply:
>java 11 was supported in AK for a while

I think it's just Java 8 features compiled with Java 11, but at any rate, since we're only using the libraries and not updating source code moving to Java 11 for tutorials should be fine. Can you create a GH issue to update all tutorials?